python/cugraph/cugraph/experimental/__init__.py

```diff
@@ -73,3 +73,7 @@
 
 overlap = experimental_warning_wrapper(EXPERIMENTAL__overlap)
 overlap_coefficient = experimental_warning_wrapper(EXPERIMENTAL__overlap_coefficient)
+
+from cugraph.gnn.data_loading import EXPERIMENTAL__BulkSampler
+
+BulkSampler = experimental_warning_wrapper(EXPERIMENTAL__BulkSampler)
```

Comment:
FYI PR #3170 changed this so that the `gnn` namespace wouldn't be lost, based on [this](https://github.com/rapidsai/cugraph/pull/3170#discussion_r1088414818)feedback.

Reply:
Yup, waiting on https://github.com/rapidsai/cugraph/pull/3170 to land before i make these changes.